### DISCUSSION TITLE
Winfom MaterialForm Drawer icon issue

### DISCUSSION THREAD

**Discussion by AliKazi5664** [Taiizor/ReaLTaiizor#143] [Q&A]
Hi, everyone ,
suddenly i'm facing problem , in design view(visual studio) everything is fine .but in runtime drawer icon(icon that click on to open the drawer) is shifted to left everything else seems to be fine,i tried check as much as i can but couldn't find anything, started new project and its working fine, unfortunately its big project and i can's start over , i attached 2 screen shot to understand what i mean exactly ,
i would really
![runtime](https://github.com/Taiizor/ReaLTaiizor/assets/146850708/a81e115b-642f-4cb1-ae26-6205ac710a27)
![designview](https://github.com/Taiizor/ReaLTaiizor/assets/146850708/7488ea9e-46d8-4d14-8ba4-d39eb6d2e5e6)
 appreciate any help or input on this

**Comment by Chaugechen** [comment]
Hi AliKazi5664
Are you looking for this kind of effect
![image](https://github.com/Taiizor/ReaLTaiizor/assets/49526083/ff73bd40-9463-459e-9616-0ca436b2d8c5)
![image](https://github.com/Taiizor/ReaLTaiizor/assets/49526083/8c103fb5-07b9-4342-aaea-83164e80c560)
